### PR TITLE
FacDB URL sources

### PR DESCRIPTION
**Pull request by mgraber**
#138 

Currently, nysoasas_programs requires specifying current date in the form dd-Mon-yy:
Example usage:
`poetry run library archive -s -c -l -v 01-Apr-21 -n nysoasas_programs`

There might be a way of formatting current date to be able to use the default version as an enhancement.

### CODE REVIEW

File: library/templates/nysdec_solidwaste.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  acl: public-read
+  source:
+    url:
+      path: https://data.ny.gov/api/views/2fni-raj8/rows.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: EPSG:4326
```
Comment (SPTKL) [inline library/templates/nysdec_solidwaste.yml:13]: is geometry type is none, set `SRS: null`, we couldn't do this before but now we can

File: library/templates/nysdec_solidwaste.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
+      type: NONE
```
Comment (SPTKL) [inline library/templates/nysdec_solidwaste.yml:20]: same as above

File: library/templates/nysdoh_healthfacilities.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  acl: public-read
+  source:
+    url:
+      path: https://health.data.ny.gov/api/views/vn5v-hh5r/rows.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: EPSG:4326
```
Comment (SPTKL) [inline library/templates/nysdoh_healthfacilities.yml:13]: SRS: null

File: library/templates/nysdoh_healthfacilities.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
```
Comment (SPTKL) [inline library/templates/nysdoh_healthfacilities.yml:19]: SRS: null

File: library/templates/nysdoh_nursinghomes.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  acl: public-read
+  source:
+    url:
+      path: https://health.data.ny.gov/api/views/izta-vnpq/rows.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: EPSG:4326
```
Comment (SPTKL) [inline library/templates/nysdoh_nursinghomes.yml:13]: SRS: null

File: library/templates/nysdoh_nursinghomes.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+      type: NONE
+
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
```
Comment (SPTKL) [inline library/templates/nysdoh_nursinghomes.yml:19]: SRS: null

File: library/templates/nysoasas_programs.yml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    url:
+      path: https://webapps.oasas.ny.gov/providerDirectory/download/Treatment_Providers_OASAS_Directory_Search_{{ version }}.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
```
Comment (SPTKL) [inline library/templates/nysoasas_programs.yml:12]: SRS: null 

and same below

File: library/templates/nysopwdd_providers.yml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    url:
+      path: https://data.ny.gov/api/views/ieqx-cqyk/rows.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: EPSG:4326
+      type: NONE
+
```
Comment (SPTKL) [inline library/templates/nysopwdd_providers.yml:15]: SRS: null

File: library/templates/nysparks_historicplaces.yml [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+  source:
+    url:
+      path: https://data.ny.gov/api/views/iisn-hnyv/rows.csv
+      subpath: ""
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+    geometry:
+      SRS: EPSG:4326
+      type: POINT
```
Comment (SPTKL) [inline library/templates/nysparks_historicplaces.yml:14]: you might have to specify x y column names for gdal to pick up geometry

**Review by AmandaDoyle — APPROVED** [review]

**Pull request merged**